Adding notes for the meeting and renaming the agenda. (#2)
kokkos/development: meeting_notes/leadership/notes/25_10_06.md

## Attendees

Christian, Damien, Luc, Siva

## Topics

#### Release 4.7.2 / 5.0.0

 - 4.7.2
   - Status, blockers?
   - For 4.7.2 we would like to update the version of Kokkos/mdspan that gets pulled in

>    - mdspan issues outstanding but we can delay the release of 4.7.2 a bit to incorporate that and not delay 5.0.

Reviewer: ```suggestion
   - mdspan issues outstanding but we can delay the release of 4.7.2 after 5.0 so we don't have to update Trilinos twice
```
